styles.css

```diff
@@ -71,25 +72,86 @@ logo {
 
 .centerbtn {
     display: flex;
-    justify-content: center; 
-    /* background-color: var(--primary-color);
-    border: none;
-    color: var(--tertiary-color);
-    width: 150px;
-    height: 40px;
-    font-size: 16px;
-    border-radius: 5px;
-    padding: 10px;  */
+    justify-content: center;
 }
 
 .footer {
     display: flex;
     align-items: center;
     padding: 10px;
-    /* background: var(--primary-color); */
-    text-decoration: none;
     color: var(--secondary-color);
     position: absolute;
     bottom: 0; 
 }
 
+.wrapper {
+    max-width: ;
+    background: var(--secondary-color);
+    box-shadow: 0 4px 4px 0 rgba(0, 0, 0, 0.05);
+    order-radius: 5px;
+    padding: 1rem;
+    margin: 1rem auto;
+    
+
+}
+
+.nav-bar2 {
+    display: flex;
+    
+}
+
+.child {
+    padding: 0px 20px;
+    flex-grow: 1;
+}
+
+.nav-link {
+    text-decoration: none;
+}
+
+
+.component-title {
+    font-size: 25px;
+    font-weight: bold;
+    color: var(--primary-color);
+    /* padding-left: 30px; */
+    margin-left: 30px;
+    padding-bottom: 10px;
+}
+
+.component-intro {
+    color: var(--primary-color);
+    /* padding-left: 30px; */
+    padding-bottom: 10px;
+    margin-left: 30px;
+}
+
+.sub-title {
+    color: var(--primary-color);
+    /* padding-left: 30px; */
+    margin-left: 30px;
+    padding-bottom: 10px;
+    font-size: 20px;
+    font-weight: bold;
+}
+
+.avatar-nav {
+    /* padding-left: 30px; */
+    margin-left: 30px;
+    margin-bottom: 20px;
+}
+
+.avatar {
+    margin-right: 15px;
+}
+
+iframe {
+    
+    margin-left: 30px;
+}
+
+hr {
+    border-color: var(--secondary-color);
+    margin: 10px 30px 10px 30px ;
```

Comment:
You can simplify this to `margin: 10px 30px`. Also it's better to use `rem` instead of `px`.

Reply:
thanks. changes updated.